src/reliability.jl

```diff
@@ -0,0 +1,77 @@
+using LinearAlgebra: Diagonal, isposdef
+
+struct Reliability
+    alpha::AbstractFloat
+    dropped::Vector{Pair{Int, AbstractFloat}}
+end
```

Comment:
I will add suggestions below to reflect this change:
```suggestion
struct Reliability{T <: Real}
    alpha::T
    dropped::Vector{Pair{Int, T}}
end
```
In particular note that `Rational` is not a subtype of `AbstractFloat` (we might add a test where covariance matrix has `Rational` `eltype`).